pkg/web/templates/partials/transaction/transaction_list.html

```diff
@@ -17,54 +18,61 @@
+          {{ if eq .Source "local" }}
+          <i class="fa-solid fa-file-export"></i>
+          {{ else if eq .Source "remote" }}
+          <i class="fa-solid fa-file-import"></i>
+          {{ else }}
+          <i class="fa-solid fa-question"></i>
+          {{ end }}
+        </td>
         {{ if .Status }}
-        <div class="flex items-center gap-x-1">
-          {{ if eq .Status "Draft" }}
+          {{ if eq .Status "draft" }}
           <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
             <div class="flex items-center gap-x-1">
               <img src="/static/draftStatusIcon.svg" alt="" />
               <span>{{ .Status }}</span>
             </div>
           </td>
-          {{ else if eq .Status "Pending" }}
+          {{ else if eq .Status "pending" }}
           <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
             <div class="flex items-center gap-x-1">
               <img src="/static/pendingStatusIcon.svg" alt="" />
               <span>{{ .Status }}</span>
             </div>
           </td>
-          {{ else if eq .Status "ActionRequired" }}
+          {{ else if eq .Status "action required" }}
           <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
             <div class="flex items-center gap-x-1">
               <img src="/static/actionStatusIcon.svg" alt="" />
               <span>{{ .Status }}</span>
             </div>
           </td>
-          {{ else if eq .Status "Completed" }}
+          {{ else if eq .Status "completed" }}
           <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
             <div class="flex items-center gap-x-1">
               <img src="/static/completeStatusIcon.svg" alt="" />
               <span>{{ .Status }}</span>
             </div>
           </td>
-          {{ else if eq .Status "Archived" }}
+          {{ else if eq .Status "archived" }}
           <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
             <div class="flex items-center gap-x-1">
               <img src="/static/archiveStatusIcon.svg" alt="" />
               <span>{{ .Status }}</span>
             </div>
           </td>
           {{ end }}
-        <td class="px-8 cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">-</td>
```

Comment:
It seemed like this was a stray extra column?

Reply:
This should have been moved up as the else in case there isn't a status.

Reply:
Oops, ok - can you make that change and merge since you're making a change to the color?